javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,41 @@
+/*console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+*/
+
+
+
+
+
+const products = getAvailableProducts();
+const ul = document.getElementById("products_list");
+const input = document.getElementById("filter_products");
+const value = input.value;
+
+input.addEventListener('input', filtered_products);
+
```

Comment:
For each key press you need to call event listener with actual value of input element.
So I would advise to start from: input.addEventListener('input', (e) => filtered_products(e.target.value));